pkg/helm/actions/release_history.go

```diff
@@ -10,6 +10,9 @@ func GetReleaseHistory(name string, conf *action.Configuration) ([]*release.Rele
 
 	history, err := client.Run(name)
 	if err != nil {
+		if err.Error() == "release: not found" {
```

Comment:
We shouldn't be comparing error messages. Is this the only way to do this? How does the console behavior change on this specific error?

Reply:
Unfortunatelly, this is the only way to do it. Helm is a CLI tool, the added check is to keep the public api and not break UI.

Reply:
How does the behavior change for this scenario?

We should really, really try to avoid looking for specific error messages.

Reply:
As as I said - without introducing this check, our Helm unit tests started to fail.

> We should really, really try to avoid looking for specific error messages.

I cannot agree more - but apparently Helm itself changed the returned messages somewhere between 3.0.x and 3.2.1. In order to avoid breaking our endpoint API, we need to catch this.

Reply:
> As as I said - without introducing this check, our Helm unit tests started to fail.

My question is what changes in the UI when this happens? If we're just showing a different error message, I would prefer just to show the error as-is from the server and not try to look for specific messages.

Reply:
@rohitkrai03 can help us here. If UI does not interpret anyhow the returned messages and display them "as is", we can simplify it then.

Reply:
@pedjak We don't do anything on the UI side to show different errors for different scenarios. UI just shows the error from the server as is.

Reply:
Thanks @rohitkrai03 In that case, we can stop interpreting the messages coming from Helm. I have pushed the simplified PR - this time we need to fix error checking in some unit tests, since some of them are now coming from 3rd party code, ie.. Helm.

/cc @spadgett @benjaminapetersen